src/app/not-found.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Link from "next/link";
+import errIcon from "../../assets/icons/404_Icon.svg";
+
+export default function NotFound() {
+  return (
+    <div className="bg-[#37526B]">
+      <div className="mx-auto w-full max-w-[1440px] pt-[90px] text-center">
+        <div className="mb-[38px]">
+          <Image
+            className="mx-auto h-[242px] w-[242px]"
+            src={errIcon}
+            alt="404 Error Icon"
+            priority
+          />
+        </div>
+
+        <div className="font-['Poppins'] text-[#FFFFFF] font-medium text-[30px] mb-[15px] leading-[30px]">
+            Page not found...
```

Comment:
The design just uses a single period. 

```suggestion
            Page not found.
```